src/components/side_panel/pivot/pivot_side_panel/pivot_side_panel_store.ts

```diff
@@ -129,7 +129,7 @@ export class PivotSidePanelStore extends SpreadsheetStore {
   }
 
   deferUpdates(shouldDefer: boolean) {
-    if (shouldDefer === false && this.draft) {
+    if (!shouldDefer && this.draft) {
```

Comment:
is it really equivalent ?

Reply:
It should be 🤔 I'll jsute revert it, that's true that it's less readable